ivy/functional/frontends/numpy/manipulation_routines/changing_number_of_dimensions.py

```diff
@@ -0,0 +1,15 @@
+# global
+import ivy
+
+
+# squeeze
+def squeeze(
+    x,
+    axis,
+    /,
+):
+    ret = ivy.squeeze(x, axis)
+    return ret
+
+
+squeeze.unsupported_dtypes = {"torch": ("float16",)}
```

Comment:
I think `float16` is supported in torch, it is `(uint16, uint32, uint64)` instead that are not supported in torch

Reply:
yes float16 supported but i think this line necessary for testing procedure. I am not using data type transfer frontend. i will remove this line in here.